proposals/incubation/openfeature.md

```diff
@@ -0,0 +1,121 @@
+# Propose OpenFeature to Incubation Stage
+
+## Introduction
+
+- Name: OpenFeature
+- Website: [openfeature.dev](https://openfeature.dev)
+- Source Control: [OpenFeature on Github](https://github.com/open-feature)
+- Code of Conduct: [OpenFeature Code of conduct](https://github.com/open-feature/community#code-of-conduct)
+- License: [Apache 2](https://github.com/backstage/backstage/blob/master/LICENSE)
+
+## Overview
+
+OpenFeature is an open standard for feature flag management, created to support a robust feature flag ecosystem using cloud native technologies. OpenFeature provides a unified API and SDK, and a developer-first, cloud-native implementation, with extensibility for open source and commercial offerings.
+
+## Sponsors from the TOC
+
+The OpenFeature project is currently looking for a TOC sponsor
+
+## Progress Since Sandbox
+
+### Community
+
+**Outreach:** As a project we hold regular community meetings each month, highlighting what is happening in the community with presentations from multiple adopters and discussions on current topics and interests in the community. Community sessions have on average **12 attendees**, are uploaded to a [Project meeting playlist hosted on Youtube](<[https://www.youtube.com/watch?v=3dV3aZo6JN8&list=PLf1KFlSkDLIBmA5TLXn2BzEHmwWzckP8y](https://www.youtube.com/playlist?list=PLiQt8D1ofl8zR0u5kzFjrX_U4rGteyOVa)>) and gained **over 750 views**.
+
+- [OpenFeature Community Hub on Github](https://github.com/open-feature/community)
+- [OpenFeature Community on Slack](https://cloud-native.slack.com/archives/C0344AANLA1)
+
+The community statistics are extracted from [CNCF devstats](https://openfeature.devstats.cncf.io/).
+
+### Technical
+
+#### SDK Progress
+Since achieveing sandbox we have progressed in the development of our SDKs:
+- [.NET](https://github.com/open-feature/dotnet-sdk) released v1.1.0
+- [Java](https://github.com/open-feature/java-sdk) released v1.2.0
+- [PHP](https://github.com/open-feature/php-sdk) released v2.0.0
+- [Python](https://github.com/open-feature/python-sdk) released v0.0.9
+- [JavaScript](https://github.com/open-feature/js-sdk) released v1.1.0
+- [GO](https://github.com/open-feature/go-sdk) released v1.2.0
+- [Ruby](https://github.com/open-feature/ruby-sdk) released v0.1.0
```

Comment:
JS is already in 1.3.1 and 1.3.2 is staged

Reply:
Most other SDKs also have newer versions, and for most of them the spec 0.6.0 update is coming.